puppet/modules/quickstack/manifests/load_balancer/neutron.pp

```diff
@@ -6,6 +6,7 @@
   $backend_server_addrs,
   $api_port         = '9696',
   $api_mode         = 'tcp',
+  $timeout         = '200s',
```

Comment:
Yes agreed! default (200s) might be large. It can be kept at current 30s but the override  should be provided. I can revert this change since you are already doing it in your BZ.